nix/gce.nix

```diff
@@ -211,6 +218,12 @@ let
     };
   };
 
+  nixosVersion = builtins.substring 0 5 (config.system.nixos.version or config.system.nixosVersion);
+
+  image-family = import ./gce-images.nix;
+  # To be changed to
+  # image-family = import <nixpkgs/nixos/modules/virtualisation/gce-images.nix>;
```

Comment:
oh you made a comment about it already :)
nitpick: image-family -> imageFamily

Reply:
Done https://github.com/nix-community/nixops-gce/pull/1/commits/fd88dea0b5a953bd15056778f95036dec0dfb2b5 , https://github.com/nix-community/nixops-gce/pull/4/commits/3322656bd49cc30103547729856f102142669768